components/AppNav.tsx

```diff
@@ -1,110 +1,85 @@
 import React, { useEffect, useState } from 'react'
 import NavLink, { NavLinkProps } from './NavLink'
 import { useRouter } from 'next/router'
-import { DropdownMenu } from './DropdownMenu'
 import { useGetAppQuery, GetAppQuery } from '../graphql'
 import Navbar from 'react-bootstrap/Navbar'
 import { Container, Nav } from 'react-bootstrap'
-import _ from 'lodash'
+import ProfileDropdownMenu from './ProfileDropdownMenu'
 import styles from '../scss/appNav.module.scss'
-import { Button } from './theme/Button'
-import LogoutContainer from './LogoutContainer'
+import {
+  REPO_PATH,
+  DOCS_PATH,
+  CURRICULUM_PATH,
+  DISCORD_PATH,
+  LOGIN_PATH,
+  SIGNUP_PATH
+} from '../constants'
+import _ from 'lodash'
 
 type AuthLinkProps = {
   session: any
 }
 
 type NavItem = NavLinkProps & { name: string }
 
-const dropdownMenuItems = [
-  { title: 'Lessons', path: '/admin/lessons' },
-  { title: 'Users', path: '/admin/users' },
-  { title: 'Alerts', path: '/admin/alerts' }
-]
 const navItems: NavItem[] = [
-  { path: '/curriculum', name: 'Curriculum' },
+  { path: CURRICULUM_PATH, name: 'Curriculum' },
   {
-    path: 'https://github.com/garageScript/c0d3-app',
+    path: REPO_PATH,
```

Comment:
Looks like this file doesn't use `/profile` anymore, so all these changes to use constants in this file is just tech debt cleanup. Can we move this to another PR? What needs to be prevented is the creation of new tech debt, which the profiledropdown file will introduce without the `/profile` constant. I'd suggest you break this PR into smaller PRs, with one being for profiledropdown menu with constants file.

Reply:
> I'd suggest you break this PR into smaller PRs, with one being for profiledropdown menu with constants file.

I think @SlyBouhafs can keep this in mind for the future. Current refactoring has no issues I think its good to merge (PRs been open for awhile).